lib/widgets/selectors.dart

```diff
@@ -100,18 +135,28 @@ List<Widget> terminalSelector(BuildContext context, bool shouldShow,
           subtitle: Text(terminal.id),
         );
       },
-      onSuggestionSelected: onSuggestionSelected,
+      onSuggestionSelected: (suggestion) {
+        onSuggestionSelected(suggestion);
+        _terminalTypeAheadController.text = suggestion.name;
+      },
     ),
   ];
 }
 
-List<Widget> orderSelector(BuildContext context, bool shouldShow,
-    void Function(Order) onSuggestionSelected) {
+List<Widget> orderSelector(
+    BuildContext context,
+    bool shouldShow,
+    void Function(Order) onSuggestionSelected,
+    TextEditingController _orderTypeAheadController) {
```

Comment:
Don't user underscore for input fields. Also the _orderType is redundant and inconsistent.  See https://github.com/bigpanther/shipanther/commit/4214f3c56159da3a102309539605062187258a26#diff-8e5e037a8700213cd43e6734b09324f63a0c7bc4c47285374f8c89ede15afc4cR18 for reference